### PR TITLE
Allow runtime script to cp the package from bin to Rpackage default path

### PR DESCRIPTION
**Pull request by dibryant**
Fixes for https://issues.redhat.com/browse/RHOAIENG-3007

## Description

Changed /opt/app-root/src/Rpackages/4.3 to /opt/app-root/bin/Rpackages/4.3

## How Has This Been Tested?

Change buildconfig on cluster to uri: 'https://github.com/dibryant/notebooks'
      ref: fix
Run the new build
Open new notebook
Install packages
Screenshot 2024-04-22 at 3 31 58 PM (2)
Screenshot 2024-04-22 at 3 40 14 PM (2)

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by harshad16** [comment]
Looks good to me.
Tested it myself.
Screenshot from 2024-04-23 15-48-16


just one note, can you make the commit message little more structure.

`Allow runtime script to cp the package from bin to Rpackage default path"

File: rstudio/rhel9-python-3.9/run-rstudio.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ fi
 
 # Create lib folders if it does not exist
 mkdir -p  /opt/app-root/src/Rpackages/4.3
+cp -r /opt/app-root/bin/Rpackages/4.3/* /opt/app-root/src/Rpackages/4.3/
```
Comment (jiridanek) [inline rstudio/rhel9-python-3.9/run-rstudio.sh:19]: What happens when user starts workbench, installs additional packages, waits few months and restarts the workbench? if this `cp` command runs then, won't it put `/opt/app-root/src/Rpackages/4.3/` into inconsistent state, mixing old and new versions of r packages?

Comment (harshad16) [reply rstudio/rhel9-python-3.9/run-rstudio.sh:19]: Yeah, good point.
perhaps, we should check, if the package already exists, we don't copy.
lets make this change.

**Comment by dibryant** [comment]
After the change I retested it and it still works with installing packages. 

Screenshot 2024-05-01 at 9 19 21 AM (2)

**Comment by dibryant** [comment]
I retested after the following change and everything is still working properly.

Screenshot 2024-05-01 at 9 30 18 AM (2)
c35478e412eb">

File: rstudio/rhel9-python-3.9/run-rstudio.sh [hunk 2]
```diff
@@ -16,6 +16,12 @@ fi
 
 # Create lib folders if it does not exist
 mkdir -p  /opt/app-root/src/Rpackages/4.3
```
Comment (atheo89) [inline rstudio/rhel9-python-3.9/run-rstudio.sh:18]: Is this intended to be created at /src?

**Comment by atheo89** [comment]
Also we should change the path on the nginx on this line here: https://github.com/dibryant/notebooks/blob/afb4eb8cdcc9c936ee2ea2cf28febdd154091da2/rstudio/c9s-python-3.9/rsession.conf#L2

**Comment by atheo89** [comment]
Tested on my end and it works as expected:

image

image

/lgtm

**Review by harshad16 — CHANGES_REQUESTED** [review]
As we want to loop over the content of the directory,
can we have the following changes please.

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

thanks :100:

**Pull request merged**